src/config/Config.cpp

```diff
@@ -277,6 +279,45 @@ void	Config::parseRoot( std::vector<std::string> &tokens )
     tokens.erase( tokens.begin() ); // Remove the ';' token.
 }
 
+void    Config::parseClientMaxBodySize( std::vector<std::string> &tokens )
+{
+    tokens.erase( tokens.begin() ); // Remove the 'client_max_body_size' token.
+
+    unsigned long number = 0;
+    while ( tokens[0] != ";")
+    {
+        int i = 0;
+        /* First, we're going to parse the numbers. We're going to do this by looping
+        through the string and checking if the current character is a digit. If it is,
+        we're going to convert it to an integer and add it to the number variable. */
+        while ( std::isdigit( tokens[0][i] ) )
+        {
+            number = number * 10 + ( tokens[0][i] - '0' );
+            i++;
+        }
+
```

Comment:
Might be nice to use std::stoi for this?
```suggestion
number = std::stoi(tokens[0], &i, 10);
```
Note that stoi throws an inception if no conversion could be performed